Automattic/pocket-casts-ios
# iOS 17 widgets: add play/pause functionality

First of all, all kudos to @jgcaruso. The code that powers this feature comes from his explorations in #921. I just cleaned up a bit and added the Pocket Casts play/pause icons. 

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/28233497-cb7f-49af-bb97-d28a4097d6f4" width="300">

## To test

1. Run the app on a device/simulator with iOS 17
2. Tap and hold on the apps screen and add the widgets: Now Playing, Up Next (medium) and Up Next (big)
3. Tap play on any episode
4. ✅ The episode should start playing and the other widgets should update accordingly
5. Pause the episode
6. ✅ The episode should pause and the other widgets should update accordingly
7. Play any other episode
8. ✅ The episode should start playing and the other widgets should update accordingly

## Checklist
